docker/lnd/Dockerfile

```diff
@@ -22,5 +22,5 @@ WORKDIR $GOPATH/src/github.com/lightningnetwork/lnd
 RUN glide install
 RUN go install . ./cmd/...
 
-COPY "docker/lnd/start-lnd.sh" .
+COPY "start-lnd.sh" .
```

Comment:
One question: does this affect the execution of the current `docker-compose` set up? If so, we need to find a solution that works for _both_ the compose set up and just docker in isolation.

Reply:
Any update on this?